Gemfile

```diff
@@ -1,7 +1,9 @@
 source 'https://rubygems.org'
 
+ruby '2.3.1'
+
 gem 'jekyll'
 gem 'github-markup'
 gem 'github-pages'
-gem 'prmd', '0.8.0'
```

Comment:
This was frozen at `0.8.0` to avoid a bug in newer versions of `prmd`.  Let's make sure to take a close look at the differences in the generated documentation.